app/app/management/commands/sync_orgs_repos.py

```diff
@@ -0,0 +1,172 @@
+                            print(f'Syncing Repo: {db_repo.name}')
+                            repo_collabs = get_repo(
+                                repo['full_name'],
+                                '/collaborators',
+                                (handle, access_token)
+                            )
+                            if 'message' in repo_collabs:
+                                print(repo_collabs['message'])
+                                continue
+                            for collaborator in repo_collabs:
+                                print(collaborator)
+                                member_user_obj = User.objects.get(profile__handle=collaborator['login'],
+                                                                   is_active=True)
+                                if member_user_obj is None:
+                                    continue
+                                member_user_profile = Profile.objects.get(handle=collaborator['login'])
+
+                                if collaborator['permissions']['admin']:
+                                    permission = "admin"
+                                elif collaborator['permissions']['push']:
+                                    permission = "write"
+                                elif collaborator['permissions']['pull']:
+                                    permission = "pull"
+                                else:
+                                    permission = "none"
+
+                                db_group = Group.objects.get_or_create(
+                                    name=f'{db_org.name}-repo-{repo["name"]}-{permission}')[0]
+                                db_org.groups.add(db_group)
+                                member_user_obj.groups.add(db_group)
+                                member_user_profile.repos.add(db_repo)
+                                print(members_to_sync)
+
+                                if collaborator['login'] not in members_to_sync or collaborator[
+                                    'login'] not in lsynced:
+                                    members_to_sync.append(collaborator['login'])
+                                for x in members_to_sync:
+                                    print(x)
+                                    try:
+                                        lsynced = lsynced + recursive_sync(lsynced, x)
+                                    except ValueError as members_loop_exec:
+                                        print("here 2")
+                                        print(members_loop_exec)
+
+                        return lsynced
+                except Exception as exc:
+                    print("here")
+                    print(exc)
+
+            for user in all_users:
+                # get profile data now creates or gets the new organization data for each user
+                try:
+                    synced = recursive_sync(synced, user.profile.handle)
```

Comment:
Getting this on my local
![image](https://user-images.githubusercontent.com/1358093/65730622-cc5fec00-e098-11e9-89cb-c6d9bbf02e1a.png)

Reply:
noted, will add some strict checking around profile having a handle value before continuing, thats strange that you don't have a profile, anytime you sync with github it should create that.

Reply:
Yes I had a user without, and also the superuser (I don't think this last one was the problem)  I had removed the one without profile because also attaching the one with a profile was getting some missing data.

Reply:
If you want to test this case you can create a user manually (one with the same name of the fixtures)

Reply:
i updated the query to get the user scoped endpoints if the new scope variable is set.